### ISSUE TITLE
Add webview to turn mobile site into an app in Android and iOS

### ISSUE THREAD

**Issue by p1slave** [binwiederhier/ntfy#625]
Here is the use case I have a website with thousands of users and the mobile version of my website is already highly optimized so it almost feels like an app except it does not support push notifications. 

My idea to solve this problem is to add a simple webview page to both Android and iOS apps so I can just enter the URL of my site and turn it into an app with push notification enabled.

Does that sound like a good idea and does anyone have time to implement this feature?

**Comment by binwiederhier** [comment]
I am confused about this idea, and what feature you'll need/want to make it possible. You want to embed ntfy into your website for push notifications? There's an example here: https://github.com/binwiederhier/ntfy/blob/main/examples/web-example-eventsource/example-sse.html -- No need to anything fancy.

That said, if you're going to use ntfy.sh for thousands of users on your website, I suggest you either host your own server, or I can host one for you.

**Comment by p1slave** [comment]
Thanks for the pointer and the offer to host a server. I'm cheap and hope a $5 DigitalOcean instance should be enough for two thousand registered users (around 150 online at peak time chatting and receiving reply notifications from posts), similar to the functionality of a Mastodon instance. I may set up a Mastodon instance too but not sure what's the estimated cost to host a ntfy server for maybe 10k users and appreciate it if you have a good estimate.

I see your link to the EventSource example and I guess the user has to keep the website open to receive the notification from ntfy but most of the time users rarely open the website in a browser and they prefer to use the app on a mobile device. It's too much of a headache for me to build a separate compatible mobile app for my web app and I want something generic like a Mastodon app where the user just enters the URL and has several Mastodon instances to switch back and forth.

Basically what I need is a browser or webview inside ntfy mobile app so my users don't have to open a Chrome tab and manually enter the URL to visit my site. It would benefit a lot of people who don't want to write a mobile app for their web app and have more users download the ntfy mobile app.
